neurom/features/__init__.py

```diff
@@ -70,6 +70,30 @@ def _fun(neurites, neurite_type=_ntype.all):
     _register_feature('NEURITEFEATURES', name, _fun, shape=(...,))
 
 
+def _find_feature_func(feature_name):
+    """Returns the python function used when getting a feature with `neurom.get(feature_name)`."""
+    for feature_dict in (NEURITEFEATURES, NEURONFEATURES):
+        if feature_name in feature_dict:
+            return feature_dict[feature_name]
+    raise NeuroMError(f'Unable to find feature: {feature_name}')
+
+
+def _get_feature_and_shape(feature_name, obj, **kwargs):
```

Comment:
Considering that we return the feature value and function, we could call it `_get_feature_value_and_func` or `_get_feature_and_func`?

Reply:
Sorry, I must be tired.